R/model_component_fwd_sim.R

```diff
@@ -1,21 +1,24 @@
 #' Function to generate a global weekly R(t) estimate with added noise
 #'
-#' @param r_in_weeks The mean R(t) for each week of the simulation, can be
-#' longer than the 7*n_weeks
+#' @param r_in_weeks The mean R(t) for each week of the simulation in natural
+#' scale, can be longer than the 7*n_weeks
```

Comment:
```suggestion
#' scale, can be longer than the 7*n_weeks
```
Missed this before. Should the 7 be here?

Reply:
whoops no good catch